pkg/util/interrupt/interrupt.go

```diff
@@ -65,7 +65,10 @@ func (h *Handler) Signal(s os.Signal) {
 func (h *Handler) Run(fn func() error) error {
```

Comment:
Updated

On Thu, Mar 31, 2016 at 11:28 AM, Rodolfo Carvalho <notifications@github.com
> wrote:

> In pkg/util/interrupt/interrupt.go
> <https://github.com/kubernetes/kubernetes/pull/23643#discussion_r58074176>
> :
>
> > @@ -65,7 +65,10 @@ func (h *Handler) Signal(s os.Signal) {
> >  func (h *Handler) Run(fn func() error) error {
>
> Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/23643/files/986cd281acd105294f723583cde2821b49cd8603#r58074176>
>